site/_posts/2017-12-18-Look-back-and-reflect.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Last Week in ManageIQ: 2017: Rear-View Mirror Reflections"
+author: AparnaKarve
+date: 2017-12-18
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+HELLO Folks! [Aparna](https://github.com/AparnaKarve) here with a special year-end edition for the 'Last Week in ManageIQ'.  Strap on your seat belts as we get into the time portal machine and review what made 2017 special.
+
+
+- The Great Split
```

Comment:
Did you consider using headings as opposed to bullet points for these?

Reply:
Sure, headings would look better.